src/github.com/getlantern/flashlight/proxied/proxied.go

```diff
@@ -387,6 +387,10 @@ func chained(rootCA string, persistent bool) (http.RoundTripper, error) {
 			ClientSessionCache: clientSessionCache,
 		},
 	}
+	if persistent {
```

Comment:
Sorry, I'm not sure I follow.  Whatever code is using this transport is free to set the Connection: Close header on new requests as it wishes, and that header will propagate through to the server.  If the server closes the connection, Transport is smart enough not to reuse it.  What's the piece that's missing?